Sources/RIBs-SwiftUI/ViewControllable.swift

```diff
@@ -17,4 +17,4 @@
 import Foundation
 import SwiftUI
 
-public protocol ViewControllable: ObservableObject {}
+public protocol ViewControllable { }
```

Comment:
Have you checked that created RIB with this changes is still build successfully? I ask it because _View_ requires _ViewModel_ to be an `ObservableObject`.

Reply:
Fixed template